stimela/commands/run.py

```diff
@@ -354,7 +355,7 @@ def run(
     parameter_file: List[str] = [],
 ):
     log = logger()
-    if not is_logging_boring():
+    if not (ctx.parent.params["boring"] or ctx.parent.params["no_progress"]):
```

Comment:
Is this a good idea? This checks the command-line arguments of the parent context, correct? Would it not be better to (a) set internal state in the parent and then (b) query that state here? The parent's CLI can change...